check.go

```diff
@@ -89,22 +89,20 @@ func (c *CheckRunner) UpdateChecks(checks api.HealthChecks) {
 		}
 		if definition.HTTP != "" {
 			http := &consulchecks.CheckHTTP{
-				Notify:        c,
-				CheckID:       checkHash,
-				HTTP:          definition.HTTP,
-				Header:        definition.Header,
-				Method:        definition.Method,
-				TLSSkipVerify: definition.TLSSkipVerify,
```

Comment:
Was TLSSkipVerify causing a problem? It should be used in http checks, right?

Reply:
Yes, it wouldn't compile with TLSSkipVerify there. I checked in consul and it had been removed.